#
# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
#
# Copyright (c) 2010-2011 Oracle and/or its affiliates. All rights reserved.
#
# The contents of this file are subject to the terms of either the GNU
# General Public License Version 2 only ("GPL") or the Common Development
# and Distribution License("CDDL") (collectively, the "License").  You
# may not use this file except in compliance with the License.  You can
# obtain a copy of the License at
# https://glassfish.dev.java.net/public/CDDL+GPL_1_1.html
# or packager/legal/LICENSE.txt.  See the License for the specific
# language governing permissions and limitations under the License.
#
# When distributing the software, include this License Header Notice in each
# file and include the License file at packager/legal/LICENSE.txt.
#
# GPL Classpath Exception:
# Oracle designates this particular file as subject to the "Classpath"
# exception as provided by Oracle in the GPL Version 2 section of the License
# file that accompanied this code.
#
# Modifications:
# If applicable, add the following below the License Header, with the fields
# enclosed by brackets [] replaced by your own identifying information:
# "Portions Copyright [year] [name of copyright owner]"
#
# Contributor(s):
# If you wish your version of this file to be governed by only the CDDL or
# only the GPL Version 2, indicate your decision by adding "[Contributor]
# elects to include this software in this distribution under the [CDDL or GPL
# Version 2] license."  If you don't indicate a single choice of license, a
# recipient has the option to distribute your version of this file under
# either the CDDL, the GPL Version 2 or to extend the choice of license to
# its licensees as provided above.  However, if you add GPL Version 2 code
# and therefore, elected the GPL Version 2 license, then the option applies
# only if the new code is made subject to such option by the copyright
# holder.
#

jdbc.invalid_connection=Invalid connection found;
jdbc.duplicate_close_connection=duplicate close() called on connection : {0}
jdbc.duplicateTxCompleted=IGNORE:In transaction completed.Probably transaction completed called again
jdbc.markedForRemoval_conClosed=RAR7111: destroying connection during connectionClosed since this connection is marked for removal
jdbc.markedForRemoval_txCompleted=RAR7112: destroying connection during txCompleted since this connection is marked for removal
jdbc.unable_to_get_client_info=RAR7114: Unable to get ClientInfo for connection
jdbc.unable_to_set_client_info=RAR7115: Unable to set ClientInfo for connection
jdbc.exc_unwrap=RAR7116: Unable to unwrap
jdbc.exc_is_wrapper=RAR7117: Exception during isWrapper
jdbc.jdbc_factory_class_load_exception=RAR7118: Unable to load jdbc objects factory
jdbc.exc_get_XA=RAR7119: Unable to get XADataSource
jdbc.exc_cce_CP=RAR7120: Unable to get ConnectionPoolDataSource
jdbc.exc_md=RAR7121: Exception while retrieving meta-data of connection
jdbc.unexpected_exception_on_detecting_jdbc_version=RAR7122: Unexpected Exception while detecting JDBC version
jdbc.config_error=RAR5048:Error in configuring JDBC resource
RAR5048.diag.cause.1=The configuration for the JDBC resource could not be read from the domain.xml
RAR5048.diag.cause.2=This is an internal server error. Please contact Sun Microsystems with the complete error log
jdbc.mbean_register=RAR5051:Jdbc pool monitoring bean registration failure
jdbc.name_lookup_failed=RAR5052:Failed to look up connection pool resource name
jdbc.exc_close=RAR5054 : Exception while closing connection
jdbc.binding=RAR5059: Binding [JDBC DataSource Name: {0}, Pool Name: {1}]
jdbc.install_resources=RAR5060: Install JDBC Datasources ...
jdbc.exc_caught_ign=RAR5061 : Exception caught and ignored : {0}
jdbc.exc_get_local_tx=RAR5062 : Exception caught while getting Local Transaction
jdbc.exc_create_xa=RAR5063 : Exception while creating XADataSource :{0}
RAR5063.diag.cause.1=The class specified in datasourceclassname attribute in the jdbc-connection-pool is not of type javax.sql.XADataSource. Please consult your database vendor's documentation to identify the class that implements the javax.sql.XADataSource
RAR5063.diag.check.1=Check if the datasourceclass implements the javax.sql.XADataSource interface. Try using the javap tool for this purpose
jdbc.exc_create_ds=RAR5065 : Exception while creating DataSource Object :{0}
RAR5065.diag.cause.1=The class specified in datasourceclassname attribute in the jdbc-connection-pool is not of type javax.sql.ConnectionPoolDataSource. Please consult your database vendor's documentation to identify the class that implements the javax.sql.ConnectionPoolDataSource
RAR5065.diag.check.1=Check if the datasourceclass implements the javax.sql.ConnectionPoolDataSource interface. Try using the javap tool for this purpose
jdbc.exc_create_ds_conn=RAR5066 : Exception while getting pooled connection :{0}
RAR5066.diag.cause.1=The Database server is not up and running
RAR5066.diag.cause.2=The connection pool is full and cannot allocate more connections
RAR5066.diag.cause.3=The Database server threw some unexpected exception. Please study the exception message for more details
jdbc.exc_create_xa_con=RAR5067 : Exception while getting XA Connection : {0}
jdbc.exc_iter=RAR5068 : No such element in connection set
RAR5068.diag.cause.1=There are no managed connections in the connection pool at this time
RAR5068.diag.cause.2=There is an internal server error. Please contact Sun Microsystems with the complete log message
RAR5068.diag.check.1=This is an internal server error. Please contact Sun Microsystems with the complete error log
RAR5068.diag.check.2=This is an internal server error. Please contact Sun Microsystems with the complete error log
jdbc.exc_eis_prodname=RAR5069 : Exception while fetching EIS Product name :{0}
RAR5069.diag.cause.1=The Connection object is now invalid due to database restart
RAR5069.diag.cause.2=The connection object is now invalid since the database is not up and running
RAR5069.diag.cause.3=The JDBC driver cannot provide the required information due to internal error
RAR5069.diag.check.1=Switch on the connection validation property of the connection pool and try again
RAR5069.diag.check.2=If the database has restarted, restart the application server as well or set the connection validation property in the pool to avoid this in the future. If the database server is not up, please bring it up
RAR5069.diag.check.3=Check the server log and please contact Sun Microsystems with the complete error log
jdbc.exc_eis_prodvers=RAR5070 : Exception while fetching EIS Product version :{0}
RAR5070.diag.cause.1=The Connection object is now invalid due to database restart
RAR5070.diag.cause.2=The connection object is now invalid since the database is not up and running
RAR5070.diag.cause.3=The JDBC driver cannot provide the required information due to internal error
RAR5070.diag.check.1=Switch on the connection validation property of the connection pool and try again
RAR5070.diag.check.2=If the database has restarted, restart the application server as well or set the connection validation property in the pool to avoid this in the future. If the database server is not up, please bring it up
RAR5070.diag.check.3=Check the server log and please contact Sun Microsystems with the complete error log
jdbc.exc_cci_constr=RAR5071 : Exception creating CciInteraction object
jdbc.exc_clr_wrn=RAR5072 : Exception while clearing warnings from Statement object
jdbc.exc_get_wrn=RAR5073 : Exception while getting warnings from Statement object
jdbc.exc_table_validation=RAR5074 : Table based validation detected invalid connection. Querying the table {0} failed. Set resource-adapter log-level to FINE for exception stack trace.
RAR5074.diag.cause.1=The Connection object is now invalid due to database restart
RAR5074.diag.cause.2=The connection object is now invalid since the database is not up and running
RAR5074.diag.cause.3=The JDBC driver cannot execute the statement due to internal error
RAR5074.diag.check.1=Switch on the connection validation property of the connection pool and try again
RAR5074.diag.check.2=If the database has restarted, restart the application server as well or set the connection validation property in the pool to avoid this in the future. If the database server is not up, please bring it up
RAR5074.diag.check.3=Check the server log and please contact Sun Microsystems with the complete error log
jdbc.exc_setup_stmt=RAR5075 : Exception while setting up Statement according to Spec
jdbc.exc_prepare_stmt=RAR5076 : Exception while preparing statement
jdbc.exc_close_stmt=RAR5077 : Exception while closing statement
jdbc.exc_beg_local_tx=RAR5078 : Exception while beginning local transaction
jdbc.exc_commit_local_tx=RAR5079 : Exception while committing local transaction
jdbc.exc_commit_xa=RAR5080 : Exception while committing xa transaction
jdbc.exc_rolbk_local_tx=RAR5081 : Exception while rolling back local transaction
jdbc.exc_cci_mc_constr=RAR5082 : Exception while creating CciManagedConnection
jdbc.exc_destroy=RAR5083 : Exception while destroying physical connection
jdbc.exc_physical_conn=RAR5084 : Exception while creating physical connection
jdbc.exc_get_xa=RAR5085 : Exception while getting XA resource
jdbc.exc_Cci_mc_md=RAR5086 : Exception creating CciManagedConnectionMetaData
jdbc.exc_eis_maxconn=RAR5087 : Exception while querying the EIS for max connections
RAR5087.diag.cause.1=The Connection object is now invalid due to database restart
RAR5087.diag.cause.2=The connection object is now invalid since the database is not up and running
RAR5087.diag.cause.3=The JDBC driver cannot provide the required information due to internal error
RAR5087.diag.check.1=Switch on the connection validation property of the connection pool and try again
RAR5087.diag.check.2=If the database has restarted, restart the application server as well or set the connection validation property in the pool to avoid this in the future. If the database server is not up, please bring it up
RAR5087.diag.check.3=Check the server log and please contact Sun Microsystems with the complete error log
jdbc.exc_del_det=RAR5088 : Exception while checking if ResultSet can detect deletes
jdbc.exc_upd_det=RAR5089 : Exception while checking if ResultSet can detect updates
jdbc.exc_other_del_det=RAR5090 : Exception while checking if ResultSet can detect other deletes
jdbc.exc_del_insert=RAR5091 : Exception while checking if ResultSet can detect inserts
jdbc.exc_other_updates=RAR5092 : Exception while checking if ResultSet can detect other updates
jdbc.exc_own_updates=RAR5093 : Exception while checking if ResultSet can detect own updates
jdbc.exc_other_ins=RAR5094 : Exception while checking if ResultSet can detect other inserts
jdbc.exc_own_ins=RAR5095 : Exception while checking if ResultSet can detect own inserts
jdbc.exc_own_deletes=RAR5096 : Exception while checking if ResultSet can detect own deletes
jdbc.exc_rs_con=RAR5097 : Exception while checking if RA supports concurrency in combo with give RS
jdbc.exc_rs_type=RAR5098 : Exception while checking if RA supports a type of RS
jdbc.exc_cnfe=RAR5100 : Wrong class name or classpath for {0}
RAR5100.diag.cause.1=Classpath is not properly set in the domain.xml
RAR5100.diag.cause.2=The application server process does not have read permissions on the directory that holds the classes/jar
RAR5100.diag.check.1=Check if the server classpath specified in the java-config element in the domain.xml has an entry for the jar containing this class
RAR5100.diag.check.2=Check that the directory where the classes/jars reside have read permission for the application server process
jdbc.exc_re=RAR5105 : Connection Error occurred : {0}
RAR5105.diag.cause.1=The connection validation method is incorrect, it should be one of auto-commit, meta-data, table
RAR5105.diag.cause.2=The connection is no longer valid. It could have been rendered invalid due to database restart/shutdown
RAR5105.diag.check.1=Check if the connection validation is one of : auto-commit, meta-data or table. If the method is table, check that the table name is correct.
RAR5105.diag.check.2=If the database has been restarted, restart the application server as well. If the database is down, start it up
jdbc.exc_autocommit_validation=RAR5106 : AutoCommit based validation detected invalid connection. Set resource-adapter log-level to FINE for exception stack trace
RAR5106.diag.cause.1=Database server is not up thus rendering the connection invalid
RAR5106.diag.cause.2=The JDBC driver is unable to provide the auto-commit status of the connection due to an internal error
RAR5106.diag.check.1=Check if the database server is up and running
RAR5106.diag.check.2=This is an internal error. Please contact Sun Microsystems with the complete error log
jdbc.exc_metadata_validation=RAR5107 : Meta-Data based validation detected invalid connection. Set resource-adapter log-level to FINE for exception stack trace
RAR5107.diag.cause.1=The connection is invalid due to a database restart
RAR5107.diag.cause.2=The connection is invalid due to a database shutdown
RAR5107.diag.cause.3=The JDBC driver cannot provide the connection's metadata information due to an internal error
RAR5107.diag.check.1=If the database has been restarted, restart the appserver as well. To avoid this in the future set the connection validation property in the pool
RAR5107.diag.check.2=Check that the database is up and running
jdbc.exc_get_tx_iso=RAR5108 : Error while getting transaction isolation level :{0}
RAR5108.diag.cause.1=The connection is invalid due to database restart/shutdown
RAR5108.diag.cause.2=The JDBC driver cannot provide the transaction isolation level due to an internal error
RAR5108.diag.check.1=Please bring up the database if not up already. Check if the connection validation pool property is set, else set this property if the database server will be restarted in the future.
jdbc.exc_tx_iso=RAR5109 : Error while setting transaction isolation level
RAR5109.diag.cause.1=Jdbc driver has thrown an SQLException while setting the isolation level.
RAR5109.diag.check.1=Check your connection pool configuration and make sure that the isolation level is correct
RAR5109.diag.check.2=Check whether your driver supports the isolation level you have specified in the connection pool.
jdbc.exc_create_mc=RAR5110 : Error creating managed Connection with user and password : {0}
RAR5110.diag.cause.1=The URL specified is incorrect
RAR5110.diag.cause.2=The database server is not up and running
RAR5110.diag.check.1=Check if the URL specified in the getConnection call is correct. Consult the JDBC driver vendor's documentation for the correct URL format.
RAR5110.diag.check.2=Check that the Database server is up and running
jdbc.exc_cce=RAR5111 : Error casting to javax.sql.Datasource
RAR5111.diag.cause.1=The class specified in datasourceclassname attribute in the jdbc-connection-pool is not of type javax.sql.DataSource. Please consult your database vendor's documentation to identify the class that implements the javax.sql.DataSource
RAR5111.diag.check.1=Check if the datasourceclass implements the javax.sql.DataSource interface. Try using the javap tool for this purpose
jdbc.exc_cce_XA=RAR5112 : Error casting to javax.sql.XADatasource
RAR5112.diag.cause.1=The class specified in datasourceclassname attribute in the jdbc-connection-pool is not of type javax.sql.XADataSource. Please consult your database vendor's documentation to identify the class that implements the javax.sql.XADataSource
RAR5112.diag.check.1=Check if the datasourceclass implements the javax.sql.XADataSource interface. Try using the javap tool for this purpose
jdbc.exc_create_conn=RAR5113 : Error creating connection from supplied information: Reason - {0}
RAR5113.diag.cause.1=The Database server is not up and running
RAR5113.diag.cause.2=The connection pool is full and cannot allocate more connections
RAR5113.diag.cause.3=The Database server threw some unexpected exception. Please study the exception message for more details
RAR5113.diag.check.1=Check that the Database server is up and running
RAR5113.diag.check.2=Check if the size of your connection pool is sufficiently large for serving all requests
RAR5113.diag.check.3=Check the server log and please contact Sun Microsystems with the full error log
jdbc.exc_get_conn=RAR5114 : Error allocating connection : [{0}]
RAR5114.diag.cause.1=There was an SQLException while acquiring connection from the Database. Please see the detailed exception message for more information
RAR5114.diag.cause.2=The Connection pool is full and incapable of serving more requests at this point. Please try later.
RAR5114.diag.cause.3=There is an internal server error. Please contact Sun Microsystems with the complete log message
RAR5114.diag.check.1=Check if the database server is correctly configured
RAR5114.diag.check.2=Check if your pool size is sufficient to server all simultaneous connection requests
RAR5114.diag.check.3=Check the server log and contact Sun Microsystems with the complete error message
jdbc.exc_cleanup=RAR5115 : Error cleaning up handles associated with this ManagedConnection
jdbc.exc_create_xa_conn=RAR5116 : The XA connection could not be allocated: {0}
RAR5116.diag.cause.1=The Database server is not up and running
RAR5116.diag.cause.2=The connection pool is full and cannot allocate more connections
RAR5116.diag.cause.3=The Database server threw some unexpected exception. Please study the exception message for more details
RAR5116.diag.check.1=Check that the Database server is up and running
RAR5116.diag.check.2=Check if the size of your connection pool is sufficiently large for serving all requests
RAR5116.diag.check.3=Check the server log and please contact Sun Microsystems with the full error log
jdbc.cannot_set_autocommit=RAR7009: Cannot set auto-commit mode for physical connection
jdbc.exc_custom_validation=RAR9000 : Custom validation detected invalid connection. Executing the isValid() of {0} class failed. Set resource-adapter log-level to FINE for exception stack trace.
RAR9000.diag.cause.1=The Connection object is now invalid due to database restart
RAR9000.diag.cause.2=The connection object is now invalid since the database is not up and running
RAR9000.diag.cause.3=The JDBC driver cannot execute the statement due to internal error
RAR9000.diag.check.2=If the database has restarted, restart the application server as well or set the connection validation property in the pool to avoid this in the future. If the database server is not up, please bring it up
RAR9000.diag.check.3=Check the server log and please contact Sun Microsystems with the complete error log
jdbc.set_custom_validation_class_name_failure=RAR9001 : Custom validation class name is invalid
RAR9001.diag.cause.1=Custom validation class name is invalid as it does not implement interface org.glassfish.jdbc.api.ConnectionValidation
RAR9001.diag.cause.2=Custom validation class name could not be loaded
RAR9001.diag.cause.3=Custom validation class name is not in the classpath.
jdbc.jdbc_factory_class_load_exception=RAR9002 : Exception while getting Proxied Connection during SQL tracing : {0}
RAR9002.diag.cause.1=Connection object could be null
RAR9002.diag.cause.2=Initialization provoked by method could have failed.
RAR9002.diag.cause.3=Underlying method inaccessible.
jdbc.sql_trace_listener_cnfe=RAR9003 : Sql tracing : ClassNotFoundException while trying to load class : {0}
RAR9003.diag.cause.1=Sql trace listener class name is not in the classpath.
jdbc.sql_trace_listener_exception=RAR9004 : Exception while introspecting SQL trace listeners : {0}
RAR9004.diag.cause.1=Constructor of the sql trace listener provided does not have enough access.
RAR9004.diag.cause.2=Sql trace listener does not implement org.glassfish.api.jdbc.SQLTraceListener.
RAR9004.diag.cause.3=Error while instantiating sql trace listener class provided.
jdbc.executing_init_sql=Executing Initialization SQL : {0}
jdbc.exc_init_sql_error=RAR9005: Error while executing Init SQL : {0}
RAR9005.diag.cause.1=Database access error
RAR9005.diag.check.1=Check is database server is up and running
jdbc.exc_init_sql_error_stmt_close=Error while closing statement at the end of Init SQL
jdbc.execute_init_sql_start=Starting Init SQL
jdbc.execute_init_sql_end=Finished executing Init SQL
jdbc-ra.timer_creation_exception=RAR9006 : Exception while creating timer : {0}
RAR9006.diag.cause.1=Bootstrap Context might not be available
jdbc.ex_get_parent_logger=RAR9007 : Exception while getting parent logger on DataSource object. Exception : {0}
RAR9007.diag.cause.1=Driver does not use java.util.logging
RAR9007.diag.cause.2=Datasource class could not be constructed because the class could not be loaded or some methods of the class could not be executed
RAR9007.diag.check.1=Driver vendor implementation should use java.util.logging to enable getting the parent logger
RAR9007.diag.check.2=Check if jdbc driver is integrated into the server.
jdbc.ex_connection_holder=RAR9008 : Exception while invoking method on Connection object. Exception : {0}
RAR9008.diag.cause.1=Database is inaccessible
RAR9008.diag.cause.2=Connection object could have been closed before this method is invoked
RAR9008.diag.cause.3=JDBC driver does not support this method
RAR9008.diag.check.1=Check if database is accessible
RAR9008.diag.check.2=Check if method is not invoked on a closed Connection object
RAR9008.diag.check.3=Check if the JDBC driver vendor provides an expected implementation
jdbc.markedForRemoval_conAborted=Destroying connection during connectionAborted since this connection is marked for removal
#------------------------------------------------------------------------------------------------------
#RAR9000 till RAR9500 reserved for jdbc-ra LogStrings.
#Please increment RAR IDs by one for any new additional messages added here.
#Add any new properties above these comments.
#------------------------------------------------------------------------------------------------------

#------------------------------------------------------------------------------------------------------
#RAR8010 till RAR8500 reserved for connectors-runtime LogStrings.
#RAR8501 till RAR8700 reserved for connectors-inbound-runtime LogStrings.
#RAR8701 till RAR8850 reserved for connectors-internal-api LogStrings.
#RAR8850 till RAR8999 reserved for work-management LogStrings.
#RAR9000 till RAR9500 reserved for jdbc-ra LogStrings.
#------------------------------------------------------------------------------------------------------
